modules/fedimint-mint-client/src/lib.rs

```diff
@@ -791,7 +791,7 @@ impl ClientModule for MintClientModule {
                             ..
                         })
                         | MintClientStateMachines::OOB(MintOOBStateMachine {
-                            state: MintOOBStates::Created(_),
+                            state: MintOOBStates::Created(_) | MintOOBStates::CreatedMulti(_),
                             ..
                         }) => Some(()),
                         _ => None,
```

Comment:
can we remove `_` case so this never happens again

Reply:
I don't like how it looks, but will do the job. Good call.